src/Module/DevSemanticLoggerProvider.php

```
@@ -0,0 +1,21 @@
+use Koriym\SemanticLogger\SemanticLogger;
+use Koriym\SemanticLogger\SemanticLoggerInterface;
+use Override;
+use Ray\Di\ProviderInterface;
+
+/** @implements ProviderInterface<SemanticLoggerInterface> */
+final class DevSemanticLoggerProvider implements ProviderInterface
+{
+    #[Override]
+    public function get(): SemanticLoggerInterface
+    {
+        return new DevSemanticLogger(new SemanticLogger());
```

PHPStan (2.1.46)

[error] Method Be\Skeleton\Module\DevSemanticLoggerProvider::get() should return Koriym\SemanticLogger\SemanticLoggerInterface but returns Koriym\SemanticLogger\DevSemanticLogger.


(return.type)